build-system/tasks/performance/measure-documents.js

```diff
@@ -70,6 +75,51 @@ const setupMeasurement = page =>
     });
   });
 
+/**
+ * Handles request interception for analytics:
+ * - rewrites vendor config requests
+ * - records first outgoing analytics request
+ *
+ * @param {Request} interceptedRequest
+ * @param {!Function} setEndTimeCallback
+ */
+async function handleAnalyticsRequests(interceptedRequest, setEndTimeCallback) {
+  const interceptedUrl = interceptedRequest.url();
+  const matchArray = interceptedUrl.match(CDN_ANALYTICS_REGEXP);
+  if (matchArray) {
```

Comment:
IMO, this is required for all tests, not only ones with analytics request handler. What do you think?

Reply:
Hmm, I don't have the same initial inclination. Can you explain your reasoning?

Reply:
My reasoning is that most pages we are testing with will include `<amp-analytics>` component, and loading the analytics vendor config would be a requirement for all test urls.

Reply:
Ok, so then are you thinking that the analyticsHandler is default included with every url?

Reply:
Nope. 
What if I'm testing with `some-url.html` with no analyticsHandler, but the document includes `<amp-analytics type='something'>`. How would you fake the vendor request in that case?

Reply:
Ooooohhhh I see what you're saying, yea totally agree. Will add it in.